cli/tests/testdata/chart1/Chart.yaml

```diff
@@ -4,8 +4,8 @@ description: A Helm chart for tobs, The Observability Stack for Kubernetes
 
 type: application
 
-version: 0.2.2
-appVersion: 0.2.2
+version: 0.3.0
```

Comment:
This is the test chart that needs to be updated to the default chart version for CI to pass.